Refine the java client doc (#516)
apache/pulsar-site: docs/client-libraries-java-use.md

}
```

> If you don't want to block your main thread and rather listen constantly for new messages, consider using a `MessageListener`. The `MessageListener` will use a thread pool inside the PulsarClient. You can set the number of threads to use for message listeners in the ClientBuilder.

Reviewer: ```suggestion
If you don't want to block your main thread but constantly listen for new messages, consider using a `MessageListener`. The `MessageListener` uses a thread pool inside the client. You can set the number of threads for message listeners in the ClientBuilder.
```
